client/signup/steps/user/index.jsx

```diff
@@ -393,6 +395,9 @@ export class UserStep extends Component {
 		) {
 			return this.props.initialContext.query.oauth2_redirect;
 		}
+		if ( this.props.initialContext.canonicalPath.includes( '/start/account' ) ) {
```

Comment:
I'm wondering if there is any reason to use `includes` rather than`startsWith`. 🤔

Reply:
The `initialContext` prop may not be provided in some environments. (e.g. unit testing) To make it safer, consider using optional chaining operators.
```js
this.props.initialContext?.canonicalPath?.includes( '/start/account' )
```